### PR TITLE
Add environment:get-parameters command

### LINKED ISSUE #47: Command to display informations about current shop state
Might be
- debug mode enable ?
- smarty cache enable ?
- memory cache enable ?
- sql configuration 
- mail configuration
- ...

### PR DESCRIPTION
**Pull request by MeKeyCool**
I am currently working on environment variables for Prestashop and I need an easy command to make my development / tests easier.

This command is under work but I let it there to receive comments or ideas for improvements ;)
If it is a problem, I can close it :)

### CODE REVIEW

**Comment by SebSept** [comment]
Thanks for contribution 🎉 

Looks fine but : 
- The command name / service name / classname must be consistent ( https://github.com/friends-of-presta/fop_console/runs/6364780235?check_suite_focus=true) - Currently there no tool to help you naming it, you should look at the existing ones to get information (#212) - (This should also be written in our contributing file).
- just a question : how does it behave it an env value is not defined ?

**Comment by MeKeyCool** [comment]
> Thanks for contribution tada
> 
> Looks fine but :
> 
>     * The command name / service name / classname must be consistent ( https://github.com/friends-of-presta/fop_console/runs/6364780235?check_suite_focus=true) - Currently there no tool to help you naming it, you should look at the existing ones to get information ([DevTool : command to generate Class name, service name from a command name. #212](https://github.com/friends-of-presta/fop_console/issues/212)) - (This should also be written in our contributing file).
> 
>     * just a question : how does it behave it an env value is not defined ?

Sorry it is still a draft but yes I will make it safier (testing if parameter exists before getting it)

Ok, I will add some comments/descriptions to the command ^^

For naming / standards, I'll try to understand and fix problem returned by CI ^^

**Comment by SebSept** [comment]
@tom-combet do you have a minute for the service name ? 
It looks correct but failed ... (I'll check too, later).

**Comment by tom-combet** [comment]
@SebSept It should be get_env instead of getenv.

**Comment by tom-combet** [comment]
@MeKeyCool Thanks for contribution!

I'll review it next week, too busy right now sry...

File: src/Commands/Environment/EnvironmentGetEnv.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        'use_debug_toolbar',
+        'cookie_key',
+        'cookie_iv',
+        'new_cookie_key',
+    ];
+
+    protected function configure(): void
+    {
+        $this->setName('fop:environment:get-env')
+            ->setAliases(['fop:environment:getenv'])
```
Comment (SebSept) [inline src/Commands/Environment/EnvironmentGetEnv.php]:
```suggestion
        $this->setName('fop:environment:get-env')
```
the alias is not needed. (it was for backward compat on other commands)

File: config/services.yml [hunk 2]
```diff
@@ -45,6 +45,10 @@ services:
     class: FOP\Console\Commands\Environment\EnvironmentDebug
     tags: [ console.command ]
 
+  fop.console.environment.getenv.command:
```
Comment (SebSept) [inline config/services.yml]:
```suggestion
  fop.console.environment.get_env.command:
```

**Comment by SebSept** [comment]
service name must be `fop.console.environment.get_env.command` (I've made a suggestion, you can commit here on github. ).

File: config/services.yml (hunk 2 above)
Comment (MeKeyCool) [reply config/services.yml]: Thx for help, I'll change this from my repos to avoid useless commits ;)

**Review by SebSept — CHANGES_REQUESTED** [review]
Thanks @MeKeyCool for you first contribution :) 
The command works fine and the parameters are in the help message, that's all good :) 

Undefined values are spotted with a red background, that good.
I suggest, you do the same for boolean values. 
Currently it's not possible to know if a configuration is `false` or an empty string, maybe you can render the boolean `true` and `false` with <note> or <comment> style.

The other point it that it's can be better to have a try/catch block to avoid unclear errors that may happen (but will probably never happen). (not mandatory, but better imo)

**Comment by MeKeyCool** [comment]
> Undefined values are spotted with a red background, that good. I suggest, you do the same for boolean values. Currently it's not possible to know if a configuration is `false` or an empty string, maybe you can render the boolean `true` and `false` with or style.
> 
> The other point it that it's can be better to have a try/catch block to avoid unclear errors that may happen (but will probably never happen). (not mandatory, but better imo)

I updated output styling.
As you let me the choice, I don't care unclear errors cause in command line I don't think this problem would be critical nor hard to solve.

Is it ok for you this way ? ^^

**Review by tom-combet — APPROVED** [review]
OK for me! Tested and approved. ✔️ 

I see one easy improvement (but not necessary, maybe in an other pull request if someone needs it): add a 'key' option to get the value of one environment variable only. Like this:

```
bin/console fop:environment:get-parameters -k "database_host"

127.0.0.1
```

**Review by SebSept — APPROVED** [review]
👍🏼 Thanks for the job and the patience !

**Pull request merged**